app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFetchScheduler.kt

```diff
@@ -53,14 +53,15 @@ class PocketVideoFetchScheduler(
             .setRequiredNetworkType(NetworkType.CONNECTED)
             .build()
 
+        // This sets the custom fetch delay for testing purposes.
+        // The custom delay will only run once.
         val delay: Long
         val workPolicy: ExistingWorkPolicy
 
-        // This sets the custom fetch delay for testing purposes.
-        // The custom delay will only run once.
         if (IS_QA_BUILD) {
-            delay = DELAY_TIME_FOR_QA_MILLIS
+            delay = checkNotNull(DELAY_TIME_FOR_QA_MILLIS) { "Fetch delay value must be set" }
             workPolicy = ExistingWorkPolicy.REPLACE
+            // Only allow this value to be used once so we do not end up spamming the Pocket servers
```

Comment:
nit: imo, I don't really like having a line of code, a line of comment, and a line of code so I'd put a newline above this or shorten the text and inline it. That being said, I don't have great reasons for that (I think it's because I like comments to "preface" the code) so I've been allowing it for other devs on the team.